packages/grafana-ui/src/components/Combobox/MultiCombobox.tsx

```diff
@@ -267,7 +267,6 @@ export const MultiCombobox = <T extends string | number>(props: MultiComboboxPro
           ))}
           {selectedItems.length > visibleItems.length && (
             <Box display="flex" direction="row" marginLeft={0.5} gap={1} ref={counterMeasureRef}>
-              {/* eslint-disable-next-line @grafana/no-untranslated-strings */}
```

Comment:
I also removed some unused disable directives that I spotted throughout

Reply:
why are these now unused? have we updated the rule to ignore non alphanumeric chars or something? 🤔